Samples/PackageWithExternalLocation/cs/PhotoStoreDemo/ExecutionMode.cs

```diff
@@ -1,9 +1,6 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Runtime.InteropServices;
 using System.Text;
-using System.Threading.Tasks;
 
 namespace PhotoStoreDemo
 {
```

Comment:
Line 25 - `result != 15700` (which I assume is `APPMODEL_ERROR_NO_PACKAGE`) is very obscure; the API returns `ERROR_SUCCESS` when successful, which is much clearer, so just look for `return == 0`.

Reply:
Also you don't need the "Is Windows 7" test - if it's not WIn7 then I am pretty sure you will crash in the JIT when it can't find `GetCurrentPackageFullName` - you need to double-wrap the function and check for `MissingMethodException` and other gymnastics. I would just ignore Win7 since it's out of support now.

Reply:
Fixed and removed.